scripts/validate.sh

```diff
@@ -1,11 +1,7 @@
 #!/bin/bash
 
 # usage: ./validate.sh bugid.bug
-# 
-# install: https://github.com/any-json/any-json 
-# install: https://github.com/jessedc/ajv-cli
-
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-SCHEMA="${DIR}/robust.json"
+VALIDATOR="${DIR}/yamale_validator.py"
 echo Checking schema compliance for $1
-any-json --input-format=yaml $1 | ajv --verbose --errors=text -s ${SCHEMA} -d /dev/stdin 2>&1 >/dev/null | sed "s/\/dev\/stdin/File $1/"
+python3 ${VALIDATOR} $1
```

Comment:
Does our CI setup guarantee Python 3 is installed?

Reply:
Not yet

Reply:
I would suggest to add that, just so we keep things working/consistent.

Just checked and the current `.travis.yml` actually specifies `2.7`.